apps/mymedicare_cb/views.py

```diff
@@ -73,27 +87,28 @@ def callback(request):
             user.email = user_info['email']
         user.save()
     except User.DoesNotExist:
-        # Create a new user. Note that we can set password
-        # to anything, because it won't be checked.
         user = User(username=user_info['sub'][9:36], password='',
                     first_name=user_info['given_name'],
                     last_name=user_info['family_name'],
                     email=user_info['email'])
         user.save()
-    up, created = UserProfile.objects.get_or_create(
-        user=user, user_type='BEN')
+
+    UserProfile.objects.get_or_create(user=user, user_type='BEN')
     group = Group.objects.get(name='BlueButton')
     user.groups.add(group)
+
     # Log in the user
     user.backend = 'django.contrib.auth.backends.ModelBackend'
     login(request, user)
+
     # Determine patient_id
     fhir_source = get_resourcerouter()
-    crosswalk, g_o_c = Crosswalk.objects.get_or_create(
+    crosswalk, _ = Crosswalk.objects.get_or_create(
```

Comment:
Any other suggestions for unused variable names? I like _ as a convention (which I picked up from Haskell...), but I'm open to other suggestions

Reply:
yup that's standard for python too as far as I know